main.js

```diff
@@ -15,7 +15,7 @@ var dialog = electron.dialog;
 
 var mainWindow = null;
 
-var debug = false;
+var debug = true;
```

Comment:
Guess you've done it to debug but should be removed from the PR.

Reply:
think i already pushed a commit to fix this https://github.com/llaske/sugarizer/pull/460/commits/ab5e5e5fc08ce39bb362ca4cb11e1a996759e26f

Reply:
Oh sorry, you're right